build/static/model-definition.json

```diff
@@ -0,0 +1,54 @@
+[
+  {
+    "id": "text",
+    "fields": [
+      {
+        "component": "text-input",
+        "name": "text",
+        "value": "Text",
+        "label": "Custom Text",
+        "valueType": "string"
+      }
+    ]
+  },
+  {
+    "id": "title",
+    "fields": [
+      {
+        "component": "select",
+        "name": "type",
+        "value": "h1",
+        "label": "Type",
+        "valueType": "string",
+        "options": [
+          { "name": "h1", "value": "h1" },
+          { "name": "h2", "value": "h2" },
+          { "name": "h3", "value": "h3" },
+          { "name": "h4", "value": "h4" },
+          { "name": "h5", "value": "h5" },
+          { "name": "h6", "value": "h6" }
+        ]
+      }
+    ]
+  },
+  {
+    "uri": "",
+    "fields": []
+  },
+  {
+    "modeltype": "container",
+    "fields": []
+  },
+  {
```

Comment:
Every model object should have an id. If there is no id, please remove the corresponding object from the array here.